test/functional/hedge.test.js

```diff
@@ -0,0 +1,223 @@
+'use strict';
```

Comment:
I think these tests should be added to https://github.com/mongodb/node-mongodb-native/blob/master/test/functional/readpreference.test.js, inside a `context('hedge', ...)` block.

Reply:
also, all of these tests use the mock server and I think should be considered unit tests. Can we test this without the mock server, either using a mock `Topology` or using command monitoring to observe the commands send to a real server?